openapi-style-guide.md

```diff
@@ -235,6 +242,39 @@ For "oauth2" security schemes, `scopes` must contain at least one entry.
 
 The key of each entry in `scopes` must be of the form "<resource URI>/scope name", where "scope name" is typically ".default" for Azure services.
 
+#### Examples
+
+The following example shows how to define a security scheme for Azure Active Directory authentication:
+```
+  "securityDefinitions": {
+    "MySecurityScheme": {
```

Comment:
I understand the name has really no significance, but shouldn't we at least use something consistent that we see in the specs repo, like "AADToken", just so it's not so jarring to see something different? All references to "MySecurityScheme" would also have to be updated, of course.